### PR TITLE
Grants form: add Nationality and Departure City questions

### PR DESCRIPTION
**Pull request by estyxx**

### CODE REVIEW

File: backend/api/grants/mutations.py [hunk 1]
```diff
@@ -68,16 +70,16 @@ def validate(self, conference: Conference, user: User) -> GrantErrors:
...
+            "travelling_from": 100,
+            "nationality": 100,
+            "departure_city": 100,
         }
         for field, max_length in max_length_fields.items():
             value = getattr(self, field, "")
 
-            if len(value) > max_length:
+            if value and len(value) > max_length:
                 print(field)
```
Comment (marcoacierno) [inline backend/api/grants/mutations.py:83]: can you remove this print?

File: frontend/src/locale/index.ts [hunk 2]
```diff
@@ -1456,9 +1462,12 @@ Non vediamo l'ora di leggere la tua storia e speriamo di vederti a PyCon Italia
     "grants.form.fields.grantType.values.diversity": "Diversity",
     "grants.form.fields.grantType.values.unemployed": "Disoccupato",
     "grants.form.fields.grantType.values.speaker": "Speaker",
-    "grants.form.fields.travellingFrom": "Da dove partirai?",
+    "grants.form.fields.travellingFrom": "Da quale paese partirai?",
     "grants.form.fields.travellingFrom.description":
-      "Indica la città e il paese da cui partirai per partecipare alla conferenza.",
+      "Seleziona il paese da cui partirai per partecipare alla conferenza.",
+    "grants.form.fields.departureCity": "Qual è la tua nazionalità?",
```
Comment (marcoacierno) [inline frontend/src/locale/index.ts:1468]: This copy seems incorrect

**Review by marcoacierno — CHANGES_REQUESTED** [review]
The italian copy is incorrect. Once fixed this should be ok to merge

File: backend/grants/models.py [hunk 3]
```diff
@@ -107,6 +107,18 @@ class ApprovedType(models.TextChoices):
         null=True,
         choices=[(country.code, country.name) for country in countries],
     )
+    nationality = models.CharField(
+        _("Nationality"),
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+    departure_city = models.CharField(
```
[the remark this answers is not in the document]
Comment (estyxx) [reply backend/grants/models.py:116]: I can change it in this PR, so we don't produce unnecessary migrations...

File: backend/api/grants/mutations.py (hunk 1 above)
Comment (estyxx) [reply backend/api/grants/mutations.py]: removed

**Review by marcoacierno — APPROVED** [review]

**Pull request merged**